examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,146 @@
+# EKS Cluster with Observability Tools
+
+This example deploy a new Kubernetes Cluster with Amazon Managed Prometheus, Amazon Managed Grafana, FluentBit, and Opensearch. It also includes instructions for deploying a sample workload with ArgoCD to generate logs and metrics.
```

Comment:
Could be reworded as:

_This example deploys a new Amazon EKS cluster with Amazon Managed Prometheus, Amazon Managed Grafana, AWS for FluentBit, and Amazon Opensearch Service. It also includes instructions for deploying a sample workload via GitOps using ArgoCD in order to generate logs and metrics._

Reply:
Also talk a bit about data flows, i.e. logs go to OpenSearch, metrics to Grafana.

Reply:
How's this?

This example demonstrates how to use the AWS EKS Accelerator for Terraform to deploy a new Amazon EKS Cluster with Amazon Managed Prometheus (AMP), Prometheus, FluentBit, and OpenSearch with Amazon Managed Grafana integration. It also provisions a sample workload with ArgoCD to generate logs and metrics.

Prometheus sends application metrics to AMP and Amazon Managed Grafana ingests data from AMP. FluentBit sends application logs to Amazon OpenSearch Service.